pages/common/lein.md

```diff
@@ -0,0 +1,23 @@
+# lein
+
+> Manage clojure projects with declarative configuration.
+
+- Generate scaffolding for a new project based on a template:
+
+`lein new {{template}} {{project_name}}`
+
+- Start a repl session either with the project or standalone:
+
+`lein repl`
+
+- Run the project's -main function with optional args:
```

Comment:
Is the `-` a typo here? Also, I think `main` may want to be enclosed in backticks.

Reply:
No, it is actually called `-main` function, I wasn't sure if backticks where used outside the command example.